### PR TITLE
Add tfsec to arkade

### LINKED ISSUE #497: Add tfsec to arkade
## Expected Behaviour
image

Add tfsec to arkade so i can use it along with the exisiting terraform
tfsec uses static analysis of your terraform templates to spot potential security issues. Now with terraform CDK support.

## Current Behaviour
No tfsec available in arkade

## Are you a GitHub Sponsor (Yes/No?)

Check at https://github.com/sponsors/alexellis

- [ ] Yes
- [x] No

### PR DESCRIPTION
**Pull request by dirien**
Signed-off-by: Engin Diri <engin.diri@mail.schwarz>

## Description

Fixes  #497 
## Motivation and Context

- [x] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))

## How Has This Been Tested?
In a fresh github codespace:
image

## Are you a GitHub Sponsor (Yes/No?)

- [ ] Yes
- [x] No

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Documentation

- [x] I have updated the list of tools in README.md if (required) with `./arkade get -o markdown`
- [ ] I have updated the list of apps in README.md if (required) with `./arkade install --help`

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

- [ ] I have tested this on arm, or have added code to prevent deployment

### CODE REVIEW

**Review by cpanato — APPROVED** [review]
tried and worked for me

File: pkg/get/get_test.go [hunk 1]
```diff
@@ -2184,3 +2184,55 @@ func Test_DownloadRekorCli(t *testing.T) {
...
+			url:     `https://github.com/aquasecurity/tfsec/releases/download/v0.57.1/tfsec-linux-amd64`,
+		},
+		{
+			os:      "linux",
+			arch:    archARM64,
+			version: "v0.57.1",
+			url:     `https://github.com/aquasecurity/tfsec/releases/download/v0.57.1/tfsec-linux-arm64`,
+		},
+		{
+			os:      "ming",
```
Comment (alexellis) [inline pkg/get/get_test.go:2220]: In a follow-up PR, let's use consts for all these repeated os strings, WDYT?

cc @yankeexe

**Review by alexellis — APPROVED** [review]
LGTM

File: pkg/get/get_test.go (hunk 1 above)
Comment (cpanato) [reply pkg/get/get_test.go:2220]: will improve this

Comment (cpanato) [reply pkg/get/get_test.go:2220]: PR: https://github.com/alexellis/arkade/pull/500

**Pull request merged**